src/object/srv_obj.c

```diff
@@ -990,31 +905,106 @@ obj_local_rw(crt_rpc_t *rpc, struct ds_cont_hdl *cont_hdl,
 	return rc;
 }
 
+/**
+ * Lookup and return the container handle, if it is a rebuild handle, which
+ * will never associate a particular container, then the container structure
+ * will be returned to \a ioc::ioc_coc.
+ */
+static int
+obj_ioc_init(uuid_t pool_uuid, uuid_t coh_uuid, uuid_t cont_uuid, int opc,
+	     struct obj_io_context *ioc)
+{
+	struct ds_cont_hdl   *coh;
+	struct ds_cont_child *coc;
+	int		      rc;
+
+	memset(ioc, 0, sizeof(*ioc));
+	rc = cont_iv_capa_fetch(pool_uuid, coh_uuid, cont_uuid, &coh);
```

Comment:
not a problem of this patch, just find that in each IO it needs to do cont_iv_capa_fetch -> create and wait ULT cont_iv_capa_refresh_ult on XS 0 as the IV cache only with one copy and avoid lock.
Seems that can cache the capa on each ds_cont_child when cont_iv_capa_ent_update @wangdi1 ?